_documentation/05-User-Manual.md

```diff
@@ -14,19 +14,126 @@ The user is relieved from most of the complexities as compared to the manufactur
 Further, many manufacturers have options that cannot be translated into IPP attributes. So the users have the option to change their printer/scanner properties through the Web GUI provided with Printer/Scanner Application.
 
 
-## Installing Applications
+## Installation
 
 For installing Printer/Scanner Applications, you should have `snap` installed on your system.
 
+### Snap
+
 To install `snap` from the Software Manager application, search for `snapd` and click Install.
 
 Alternatively, `snapd` can be installed from the command line:
 
     sudo apt install snapd
 
-Once `snap` is installed, you can easily install any driver with ease. Find the name of the application compatible with your device. This information will be provided by your device manufacturer. You can install the application from the command line using the command:
+### Applications
 
-    snap install <application-name>
+Once `snap` is installed, you can easily install any driver. Find the name of the application compatible with your device. This information will be provided by your device manufacturer. You can install the application from the command line:
 
+    snap install <application-name>
 
+<br>
 *Note: Driver auto-installation i.e. when a printer on the system is discovered, the driver can be found on the Snap Store and even can be automatically installed using the hardware signature, is under development. We will update this work on our website as soon as this is completed. Until then you have to install drivers manually.*
+
+## Web Interface
+
+The embedded server can also provide a web interface to the Printer/Scanner Application, and PAPPL includes a standard web interface that can be customized and/or overridden. Aside from the usual status monitoring functionality, the web interface can be configured to allow remote users (with proper authentication) to:
+
+* Create and delete printers
+
+* Set the printer location and DNS-SD name
+
+* Configure the loaded media
+
+* Configure remote access accounts
+
+* Configure networking settings such as the hostname 
+
+* Update the TLS certificates used by the server
+
+![Webp net-resizeimage (1)](https://user-images.githubusercontent.com/43112419/97160934-1850f900-17a3-11eb-93cd-fe881c5816b7.png)
+
+___
+
+### Create and Delete Printers
+
+* **Create printers**
+
+    Resource path: /addprinter
+
+    ![Webp net-resizeimage (4)](https://user-images.githubusercontent.com/43112419/97161428-d2e0fb80-17a3-11eb-856b-b56752f4995e.png)
+
+    Addition of printer requires only the following attributes:
+    1. **Name** of the printer
+    2. **Device** can be one from the following:
+        * [Label Printer](https://en.wikipedia.org/wiki/Label_printer)
+        * Office Printer
+        * [Network Printer](https://www.computerhope.com/jargon/n/netwprin.htm#:~:text=A%20network%20printer%20is%20a,to%20a%20single%2C%20dedicated%20computer.)
```

Comment:
The choices here vary with the Printer Application and the available printer devices. Usually here you will see printers which were discovered on your system, both local (USB and also manufacturer-specific or legacy connection types) and network printers. ideally only the ones actually supported by this Printer Application. Sometimes also manual options, like "Network Printer" with the possibility to eneter a host name/IP in the next field, are available, too.